appveyor.yml

```diff
@@ -0,0 +1,23 @@
+environment:
+  matrix:
```

Comment:
Ugh, I can think of many things that can get out of sync now that we need to maintain two CI config files. Most of the things will result in a build failure which is fine but some things will not. Changing Node versions is one of them. I suggest:

* Add a comment to the version list in `.travis.yml` saying: "Keep this in sync with appveyor.yml"
* Add another comment above the version list here saying "Keep this in sync with .travis.yml"